opendebates/prefixed_urls.py

```diff
@@ -1,43 +1,31 @@
 from django.conf.urls import include, url
 
+from opendebates import views, moderator_views
 
 urlpatterns = [
-    url(r'^$', 'opendebates.views.list_ideas', name="list_ideas"),
-    url(r'^changelog/$', 'opendebates.views.changelog', name='changelog'),
+    url(r'^$', views.list_ideas, name="list_ideas"),
+    url(r'^changelog/$', views.changelog, name='changelog'),
 
-    url(r'^recent/$', 'opendebates.views.recent_activity',
-        name='recent_activity'),
-    url(r'^questions/(?P<id>\d+)/vote/$', 'opendebates.views.vote', name="vote"),
-    # url(r'^comment/$', 'opendebates_comments.views.post_comment',
-    #     name="comment"),
-    url(r'^questions/(?P<id>\d+)/$', 'opendebates.views.vote', name="show_idea"),
-    url(r'^questions/(?P<id>\d+)/report/$', 'opendebates.views.report',
-        name='report'),
-    url(r'^questions/(?P<id>\d+)/merge/$', 'opendebates.views.merge',
-        name='merge'),
+    url(r'^recent/$', views.recent_activity, name='recent_activity'),
+    url(r'^questions/(?P<id>\d+)/vote/$', views.vote, name="vote"),
+    # url(r'^comment/$', 'opendebates_comments.views.post_comment', name="comment"),
```

Comment:
Can we either update this (in case someone uncomments it some day) or just delete the commented line if we're never going to need it?

Reply:
It looks like this line is referencing a view that hasn't existed for a few years, so I'm going to just remove it.